<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskCreationDialog</class>
 <widget class="QDialog" name="TaskCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Binarificator Task</string>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Input File:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="inputFileLayout">
     <item>
      <widget class="QLineEdit" name="inputFileLine"/>
     </item>
     <item>
      <widget class="QToolButton" name="inputFileBrowseButton">
       <property name="toolTip">
        <string>Browse</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output File:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="outputFileLayout">
     <item>
      <widget class="QLineEdit" name="outputFileLine"/>
     </item>
     <item>
      <widget class="QToolButton" name="outputFileBrowseButton">
       <property name="toolTip">
        <string>Browse</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Font:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="fontButton">
     <property name="text">
      <string>Consolas, 10pt</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="selectedFont" stdset="0">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="backgroundColorButton">
     <property name="text">
      <string>rgba(0, 0, 0, 255)</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="selectedBackgroundColor" stdset="0">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Output Size:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string>Warning: High values may cause your entire operating system to lag or even freeze entirely. The developer is not responsible for any damages taken.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="multiplySizeRadioButton">
            <property name="text">
             <string>Multiply Input File's...</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="_2">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="multiplyWidthByLabel">
              <property name="text">
               <string>Width By:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="multiplyHeightByLabel">
              <property name="text">
               <string>Height By:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QDoubleSpinBox" name="multiplyHeightBySpinBox">
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="decimals">
                 <number>6</number>
                </property>
                <property name="minimum">
                 <double>0.000001000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QDoubleSpinBox" name="multiplyWidthBySpinBox">
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="decimals">
                 <number>6</number>
                </property>
                <property name="minimum">
                 <double>0.000001000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="customOutputSizeRadioButton">
            <property name="text">
             <string>Custom</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="_3">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="customWidthLabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QDoubleSpinBox" name="customWidthSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="suffix">
                 <string>px</string>
                </property>
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999999999999.000000000000000</double>
                </property>
                <property name="value">
                 <double>256.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="customHeightLabel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QDoubleSpinBox" name="customHeightSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="accelerated">
                 <bool>true</bool>
                </property>
                <property name="suffix">
                 <string>px</string>
                </property>
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999999999999.000000000000000</double>
                </property>
                <property name="value">
                 <double>256.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="addTaskButton">
     <property name="text">
      <string>Add Task</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
